Bio/EBI/ebisearch.py

```diff
@@ -0,0 +1,798 @@
+# Copyright 2017 by Berenice Batut (berenice.batut@gmail.com). All rights reserved.
+# Revision copyright 2017 by Francesco Gastaldello. All rights reserved.
+#
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+
+"""Provide code to access the EBI Search API.
+
+This module aims to make EBI Search API easy to use. See:
+https://www.ebi.ac.uk/ebisearch/overview.ebi
+
+The EBI Search API is a text search engine to access biological data resources
+hosted at the European Bioinformatics Institute (EMBL-EBI).
+
+The following code was developed by Berenice Batut (berenice.batut@gmail.com)
+
+References:
+Park Y.M., Squizzato S., Buso N., Gur T., Lopez R. (2017)
+The EBI search engine: EBI search as a service making biological data accessible
+for all.;Nucleic Acids Research, May 2, 2017; DOI: 10.1093/nar/gkx359
+
+"""
+
+from Bio._py3k import urlopen as _urlopen
+from Bio._py3k import Request
+import json
+
+
+base_url = 'http://www.ebi.ac.uk/ebisearch/ws/rest'
+size_limit = 100
+start_limit = 250000
+
+
+def get_domain_details(domain):
+    """Return a dictionary with the details of a given domain in EBI.
+
+    Arguments:
+     - domain - domain id in EBI.
+
+    Example:
+
+    >>> dom_dict = ebisearch.get_domain_details(domain="allebi")
+    >>> print(str(dom_dict['domains'][0])[:150])
+    >>> {u'description': u'Results from all the EBI resources', u'subdomains':
+        [{u'description': u'', u'subdomains': [{u'fieldInfos': [{u'description'
+        : u'Luce
```

Comment:
The indentation is wrong, if this is to be a working doctest the output of the print statement it should be exact - and therefore truncated at 75 might be better than 150. Right now there also looks to be a comma or something missing before the "Luce" entry.

Reply:
Phaving a mixture of doctest and unittest is not a good idea. 
I could remove this part here and test the function in `Tests/test_EBI_ebisearch.py`?